aep/general/0111/aep.md

```diff
@@ -0,0 +1,66 @@
+# Planes
+
+Resources and methods on an API can be divided into the _plane_ that they
+reside or perform operations upon. For the context of APIs, the following
+planes are defined:
+
+- Management plane: a uniform, resource-oriented API that primarily configures
+  and allows retrieval of resources.
+- Data plane: a heterogenous API (ideally resource-oriented) that reads and
+  write user data. Often connects to entities provisioned by the management
+  plane, such as virtual machines.
+
+The term "plane" was originally used in networking architecture. Although
+system and network architecture often defines additional planes (e.g. control
+plane or power planes), as the AEPs are focused on the interface, they are not
+defined in this AEP.
+
+## Guidance
+
+### Management Plane
+
+Management resources and methods exist primarily to provision, configure, and
+audit the resources that the data plane interfaces with.
+
+As an example, the following are considered management resources for a cloud
+provider:
+
+- virtual machines
+- virtual private networks
+- virtual disks
+- a blob store instance
+- a project or account
+
+### Data Plane
+
+Methods on the data plane operate on user data in a variety of data formats,
+and generally interface with a resource provisioned via a management plane API.
+Examples of data plane methods include:
+
+- writing and reading rows in a table
+- pushing to or pulling from a message queue
+- uploading blobs to or downloading blobs from a blob store instance
+
+Data plane APIs **may** be heterogenous across a larger API surface, due to
```

Comment:
What does it mean for an API to be heterogenous?  Not entirely resource-oriented?

What is a "larger API surface"?  Larger than what?

Reply:
`across a larger API surface` doesn't seem helpful here - will remove.

I probably can be more precise that heterogeneity - I mean that there's more leeway in the requirements of data plane vs those for management APIs, up to being completely non-AEP compliant (e.g. SQL or a protocol for blob storage).